core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -754,17 +754,41 @@ class AutoTuner(
       limitedLogicList: Option[Seq[String]] = Some(Seq("spark.sql.shuffle.partitions")),
       showOnlyUpdatedProps: Boolean = true):
       (Seq[RecommendedPropertyResult], Seq[RecommendedCommentResult]) = {
-    filterByUpdatedPropertiesEnabled = showOnlyUpdatedProps
-    limitedLogicList.foreach { limitedSeq =>
-      limitedSeq.foreach(_ => limitedLogicRecommendations.add(_))
-    }
-    skipList.foreach(skipSeq => skipSeq.foreach(_ => skippedRecommendations.add(_)))
-    if (processPropsAndCheck) {
-      initRecommendations()
-      calculateRecommendations()
+      val isPluginLoaded = getPropertyValue("spark.plugins") match {
+        case Some(f) => f.contains("com.nvidia.spark.SQLPlugin")
+        case None => false
+      }
+      val rapidsEnabled = getPropertyValue("spark.rapids.sql.enabled") match {
+        case Some(f) => f.toBoolean
+        case None => true
+      }
+    if (!isPluginLoaded || !rapidsEnabled) {
+      appendComment("AutoTuner recommendations only support " +
+          "eventlogs generated by Spark applications\n  running on GPU hardware with Spark " +
+          "RAPIDS Plugin enabled.")
+      if (!isPluginLoaded) {
+        appendComment("The Plugin jar is missing from \"spark.plugins\". Please refer " +
```

Comment:
- Instead of inserting newlines, we can make those separate comments
- "Plugin jar" is ambiguous.
- There is missing punctuation missing before "using the GPU plugin then enable Spark RAPIDS"
- suggesting rephrasing to avoid ambiguity of "plugin": The RAPIDS Accelerator for Apache Spark is not included in the \"spark.plugins\" list.

Reply:
Thank you. I was struggling with that while I was making the changes. Having each of the suggestions as a separate comment makes more sense. 

I have made the changes.